cattle/plugins/host_info/iops.py

```diff
@@ -0,0 +1,50 @@
+import logging
+import platform
+import json
+
+log = logging.getLogger('iops')
+
+
+class IopsCollector(object):
+    def __init__(self):
+        self.data = {}
+
+    def _get_iops_data(self, read_or_write):
+        with open('/var/lib/rancher/state/' + read_or_write + '.json') as f:
+            return json.load(f)
+
+    def _parse_iops_file(self):
+        data = {}
+
+        try:
+            read_json_data = self._get_iops_data('read')
+            write_json_data = self._get_iops_data('write')
+        except IOError:
+            # File doesn't exist. Silently skip.
+            return {}
+
+        read_iops = read_json_data['jobs'][0]['read']['iops']
```

Comment:
do we need to check array length value here? or is it safe to assume that it's always > 0